src/mcp_agent/workflows/swarm/swarm.py

```
@@ -84,8 +84,8 @@ def __init__(
         self,
         name: str,
         instruction: str | Callable[[Dict], str] = "You are a helpful agent.",
-        server_names: list[str] = None,
-        functions: List["AgentFunctionCallable"] = None,
+        server_names: list[str] = [],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)